README.md

```diff
@@ -1,15 +1,93 @@
 # Temporal Ruby SDK
 
-![CI status](https://github.com/temporalio/sdk-ruby/actions/workflows/ci.yml/badge.svg?branch=main)
+[![CI status](https://github.com/temporalio/sdk-ruby/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/temporalio/sdk-ruby/actions/workflows/ci.yml?query=branch%3Amain)
+[![Build](https://github.com/temporalio/sdk-ruby/actions/workflows/build-native-ext.yml/badge.svg)](https://github.com/temporalio/sdk-ruby/actions/workflows/build-native-ext.yml?query=branch%3Amain+event%3Arelease)
+[![Gem Version](https://badge.fury.io/rb/temporalio.svg)](https://rubygems.org/gems/temporalio)
 
-Temporal is a distributed, scalable, durable, and highly available orchestration engine used to
-execute asynchronous long-running business logic in a scalable and resilient way.
+[Temporal](https://temporal.io/) is a distributed, scalable, durable, and highly available
+orchestration engine used to execute asynchronous long-running business logic in a scalable and
+resilient way.
 
 "Temporal Ruby SDK" is the framework for authoring workflows and activities using the Ruby
 programming language.
 
 ⚠️ UNDER DEVELOPMENT
 
+The Ruby SDK is under development. There are no compatibility guarantees at this time.
+
+At this point the SDK only supports the **Temporal Client** capabilities:
```

Comment:
Might be worth adding to the "limited capability warnings" that usage on Temporal Cloud is not yet supported (ie. because no mTLS support at this time).